docs/src/main/asciidoc/vision.adoc

```diff
@@ -28,13 +33,38 @@ dependencies {
 }
 ----
 
-=== Cloud Vision Template
+==== Cloud Vision OCR Dependencies
+
+If you are interested in applying optical character recognition (OCR) on documents for your project, you'll need to add both `spring-cloud-gcp-starter-vision` and `spring-cloud-gcp-starter-storage` to your dependencies.
+The storage starter is necessary because the Cloud Vision API runs OCR on documents stored in Google Cloud Storage buckets.
```

Comment:
Isn't it also that it stores the output of the OCR process in the GCS bucket?

Reply:
Done.